hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/libhdfs/jni_helper.c

```diff
@@ -487,10 +488,10 @@ static ssize_t wildcard_expandPath(const char* path, char* expanded)
  * allocated after using this function with expandedClasspath=NULL to get the
  * right size.
  */
-static ssize_t getClassPath_helper(const char *classpath, char* expandedClasspath)
+static x_platform_ssize_t getClassPath_helper(const char *classpath, char* expandedClasspath)
 {
-    ssize_t length;
-    ssize_t retval;
+    x_platform_ssize_t length;
```

Comment:
I put the `x_platform_` prefix to avoid any possible future conflicts with ssize_t. Like, if someone were to include a third-party library header that defines its own version of ssize_t, it shouldn't affect these parts.
That's just my view. However, I don't mind just using the ssize_t typedef instead of x_platform_ssize_t if you prefer that.

Reply:
In some other file, we use the namespace suffix, that looks a little bit more maintainable than a full new type.

Reply:
I've defined `ssize_t ` type under the `XPlatform::` namespace in C++ files. Since C lacks namespaces, I had to resort to adding `x_platform_` prefix.
 
But no worries, even I had some reservations with this approach. I'll just rewrite this PR to use a common `ssize_t` typedef in a header file. We'll worry about the future possible conflicts with `ssize_t` if/whenever it happens.